src/frequenz/sdk/actor/_actor.py

```diff
@@ -153,5 +153,7 @@ def cancel(self, msg: str | None = None) -> None:
         Args:
             msg: The message to be passed to the tasks being cancelled.
         """
+        if msg is not None:
```

Comment:
Isn't this calling the super method and then its logged twice?

Reply:
In Actor has info log.
In Backgroud has debug log.

Having info log in BackgroundService will create lots of unnecessary logs (that we probably don't need).


Do you have different idea?

Reply:
Ah, well in this case it seems fine then. Didn't notice the different levels

Reply:
Cancel sounds like a rare event, and it's useful to know. INFO doesn't trigger any alerts. It wouldn't be too bad if we just had INFO in the background service.

Reply:
Oh, I didn't see this comment. For me this still sounds like a debug, because I guess it is only useful when debugging, when things are working you should see an INFO for the actor stopping, and that should be enough.